code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/internal/rancher/RancherIdentitySearchProvider.java

```diff
@@ -94,7 +93,7 @@ public Identity getIdentity(String id, String scope) {
 
     @Override
     public boolean isConfigured() {
-        return !SecurityConstants.SECURITY.get();
+        return true;
```

Comment:
The change is so that when auth is on a request still gets a rancher_id with its set of identities in the policy. Without that identity it cannot access projects that are rancher_id member only which breaks apiKeys ability to access projects and do things they might want / need to be able to do.